### ISSUE TITLE
Anyone interested in participating in documenting a model of how ad tech works for use within Fledge

### ISSUE THREAD

**Issue by brodrigu** [brodrigu/fledge-ad-tech-model#1]
Please comment if you are interested.

**Comment by fbastello** [comment]
Happy to participate!

**Comment by drpaulfarrow** [comment]
Yep. Definitely happy to participate. Sure Julien Delhommeau will be interested too, but he's out until September.

**Comment by JoelPM** [comment]
I put this together a while ago. It might help (or maybe not):
https://docs.google.com/presentation/d/1q9bES3x5NrDqR1Qa-luQrPx3LZenx3TqWu6MPrB1gYE

**Comment by drpaulfarrow** [comment]
Great slides, Joel. Definitely form a nice starting point. 

I personally don't think we need to justify the existence of multiple SSPs for this specific ask. The fact is, multiple SSPs exist, driven by market demand, and any browser framework should recognise that and allow for/accommodate it.

How do we want to proceed? I can take a bash at whittling these slides down to like, I dunno, 3?

**Comment by drpaulfarrow** [comment]
Can we link up on Slack easily enough? Might help facilitate discussion.

**Comment by JoelPM** [comment]
I appreciate the feedback, @paulandrewfarrow. Those slides were put together a couple months ago when there were questions about why multiple SSPs are even a thing. I've rearranged the deck to put the commentary in appendices at the end and leave the actual proposed solution(s) in the first few slides. I would really like to see a solution like that discussed on pages 5-8.

**Comment by brodrigu** [comment]
I think I setup a mutli-org slack channel, please try this out: https://join.slack.com/share/zt-urz762ox-cZH8h_cQMkZ4CPbmtediDw

**Comment by vincent-grosbois** [comment]
Hello @brodrigu 
I would be interested too !
